config/software/ruby.rb

```diff
@@ -32,6 +32,16 @@
 
 fips_enabled = (project.overrides[:fips] && project.overrides[:fips][:enabled]) || false
 
+# Workaround for broken Omnibus OpenSSL config.  The OpenSSL URL
+# changed, but the Omnibus config still has the wrong URL.  See
+# https://github.com/chef/omnibus-software/issues/833.
+#
+# TODO(davidbtucker): Remove once
+# https://github.com/chef/omnibus-software/pull/1178 is merged.
+project.override :openssl,
+                 source: {url: "https://www.openssl.org/source/old/1.0.2/openssl-1.0.2t.tar.gz", extract: :lax_tar},
+                 relative_path: "openssl-1.0.2t"
```

Comment:
Just curious, why not `1.0.2u`?

Reply:
1.0.2t is the version it was trying to download but failing.

I'm not clear on how Omnibus determines the actual version to use though, so there might be a better solution.

Reply:
You're right — `1.0.2t` is the default version for the release of `omnibus-software` that we're using (https://github.com/chef/omnibus-software/blob/433a18fa1fe9690ad939e570f301771a804d0cf8/config/software/openssl.rb#L27).